src/main/java/gov/usgs/wma/waterdata/collections/CollectionsController.java

```diff
@@ -25,14 +29,20 @@
 @Tag(name = "Observations - OGC api", description = "Feature Collections")
 @RestController
 public class CollectionsController {
+	private static final Logger log = LoggerFactory.getLogger(CollectionsController.class);
```

Comment:
Looks like log is just being used once to print a trace statement. Probably could be cleaned up

Reply:
I am going to check to make sure that is not in the most recent commit. I had added testing logging statements to figure out where things where breaking early on.

Reply:
That said, on second thought, that is why we use logging, right? So we do not have to "clear it up" that we can leave it in there unlike printlns. On that note, 12 Factor says we should pretty much go back to println (console logging) so that logs are in one place. Bell bottoms anyone?

Reply:
I can see that. It is a balance, at some point the extra log statements become clutter versus
the likelihood that they will be used in the future. This might not be the tracing
we would want to use anyway.